R/constants.R

```diff
@@ -31,6 +32,14 @@ GTAP_HISTORICAL_YEAR <- 2000
 aglu.GLU <- "GLU"
 aglu.GLU_NAME_DELIMITER <- ""  # delimiter between the GLU name and number
 
+# cotton seed and lint conversion
+CONV_COTTON_LINT <- 0.4
```

Comment:
Are there sources for these conversions? And maybe a little more information on what we are converting?